modules/sdk-coin-sui/src/sui.ts

```diff
@@ -791,4 +793,9 @@ export class Sui extends BaseCoin {
 
     return { transactions: consolidationTransactions, lastScanIndex };
   }
+
+  /** inherited doc */
+  setCoinSpecificFieldsInIntent(intent: PopulatedIntent, params: PrebuildTransactionWithIntentOptions): void {
+    intent.unspents = params.unspents;
+  }
```

Comment:
just checking, ticket only says ADA; is this expected for SUI too ?

Reply:
I've just refactored it. [WIN-3106](https://bitgoinc.atlassian.net/browse/WIN-3106)